pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -91,30 +91,29 @@ func NewUncertainProgressError(msg string) *UncertainProgressError {
 // newCsiDriverClient.
 func newV1NodeClient(addr draAddr) (nodeClient drapbv1.NodeClient, closer io.Closer, err error) {
 	var conn *grpc.ClientConn
+
 	conn, err = newGrpcConn(addr)
 	if err != nil {
 		return nil, nil, err
 	}
 
-	nodeClient = drapbv1.NewNodeClient(conn)
-	return nodeClient, conn, nil
+	return drapbv1.NewNodeClient(conn), conn, nil
 }
 
 func NewDRAPluginClient(pluginName string) (DRAClient, error) {
 	if pluginName == "" {
-		return nil, fmt.Errorf("plugin name is empty")
+		return nil, errors.New("plugin name is empty")
```

Comment:
There's nothing technically wrong with using fmt.Errorf here. I'd rather be consistent on all errors.